packages/sdk/src/fee/feeOracle.ts

```diff
@@ -141,7 +142,12 @@ export const requestFeeFromFeeOracle = async ({
 }) => {
   try {
     const response = await fetch(
-      `${feeOracleBaseUrl}/v1/rate/from/${fromDomainID}/to/${toDomainID}/token/${resourceID}`,
+      `${feeOracleBaseUrl}/v1/rate/from/${fromDomainID}/to/${toDomainID}/resourceid/${resourceID}`,
```

Comment:
it seems here there is an issue with eslint. There is a new line before the string

Reply:
It looks like github's highlight is a bit misleading here. 
`/token/` changed to `/resourceid/` at the end of the line